bsip-0061.md

```diff
@@ -0,0 +1,53 @@
+    BSIP: 0061
+    Title: Fix locked accounts with circular dependencies 
+    Authors: OpenLedgerApp <https://github.com/OpenLedgerApp>
+    Status: Draft
+    Type: Protocol
+    Created: 2019-22-02
+    Updated: 
+    Discussion: https://github.com/bitshares/bsips/issues/94
+    Worker:
+
+
+# Abstract
+BitShares allow to change permissions of account A to account A. See *Cycles* paragraph of https://bitshares.org/technology/dynamic-account-permissions for details. More information can be found here: https://steemit.com/blockchain/@hipster/sad-story-how-i-lost-bitshares-account
+
+# Motivation
+We offer to develop additional functionality "Prevent to create cycled accounts". This functionality will avoid to create or update account authority with circular dependencies in the future.
+
+# Rationale
+According to our investigation, BitShares has about 384 locked accounts with circular dependencies. Most of these accounts are locked by mistake. 
+
+# Specifications
+
+## Prevent circular dependencies
+We must extend create and update account evaluators adding code that detect cycled authorities and throws exception in do_evaluate() method if cycle detected.
```

Comment:
This is the best part so far. Within the Specifications section we typically see some pseudocode blocks. Please take guidance from others on what level of code/logic detail should be included.

Reply:
Done. Updated Rational and Specifications